private.tf

```diff
@@ -5,21 +5,25 @@ module "private_label" {
   name      = "${var.name}-private"
 }
 
-resource "aws_subnet" "private" {
-  count = "${length(var.availability_zones)}"
+resource "null_resource" "private_cidr_block" {
```

Comment:
Define 1 `null_resource`. Define 2 triggers: `private_cidr_block` and `public_cidr_block`.

Reply:
Nevermind, I see why you separated them since they are in two separate files. I'm okay with this.